stingray/lightcurve.py

```diff
@@ -94,6 +103,10 @@ def __init__(self, time, counts, input_counts=True):
 
         self.tseg = self.time[-1] - self.time[0] + self.dt
         self.tstart = self.time[0] - 0.5*self.dt
+        self.gti = \
+            np.asarray(assign_value_if_none(gti,
+                                            [[self.tstart,
+                                              self.tstart + self.tseg]]))
```

Comment:
Wait, so the GTI is never actually applied to the data? In this case, would it be useful to have a method `apply_gti` that spits out, say, a list of light curves of contiguous segments for which he GTIs have been applied?